docs/getting-started/index.mdx

```diff
@@ -0,0 +1,162 @@
+Region                        United States (us)
+Latency                       78ms
+Web Interface                 http://127.0.0.1:4040
+Forwarding                    https://84c5df474.ngrok-free.dev -> http://localhost:8080
+
+Connections                   ttl     opn     rt1     rt5     p50     p90
+                              0       0       0.00    0.00    0.00    0.00
+```
+
+Open the Forwarding URL in your browser and you will see your web application.
+
+- This URL is available to anyone on the internet. Test it out by sending it to
+  a friend!
+- Your app is available over HTTPS (notice the 🔒 in your browser window) with
+  a valid certificate that ngrok automatically manages for you.
+
+## Step 4: Always use the same domain
+
+If you want to keep the same URL each time you use ngrok, [create a static
+domain on your dashboard](https://dashboard.ngrok.com/cloud-edge/domains) and
+then use the `--domain` flag to ask the ngrok agent to use it. First, stop
+ngrok with `Ctrl+C` and then run ngrok again:
+
+```
+ngrok http 8080 --domain jumpy-red-mollusk.ngrok-free.app
+```
+
+## Step 5: Secure your app
+
+You may not want everyone to be able to access your application. ngrok can
+quickly add authentication to your app without any changes. If your Google
+account is `alan@gmail.com`, you can restrict access only for yourself by
+running ngrok with:
+
+```
+ngrok http http://localhost:8080 --oauth=google --oauth-allow-email=alan@gmail.com
+```
+
+Anyone accessing your app will be prompted to log in with Google and only your
+account will be allowed to access it. Keep in mind that when you restart ngrok,
+if you don't specify the `--domain` flag that your app's URL changed, so make
+sure to visit the new one.
+
+<img width="400" height="400" src="/docs/img/docs/ngrok-oauth.png" />
+
+If you don't have a Google account or you want a simpler form of auth, you can
+protect your app with a username and password like so:
+
+```
+ngrok http http://localhost:8080 --basic-auth 'username:a-very-secure-password'
+```
+
+<img width="450" height="450" src="/docs/img/docs/ngrok-basic-auth.png" />
+
+ngrok supports many forms of authentication including:
+
+- [OAuth](/http/oauth/) (what we just used)
+- [Basic Auth](/http/basic-auth/) (what we just used)
+- [IP Restrictions](/http/ip-restrictions/)
+- [Webhook Verification](/http/webhook-verification/)
+- [Mutual TLS](/http/mutual-tls/)
+- [OpenID Connect](/http/openid-connect/)
+- [SAML](/http/saml/)
+
+## What's next?
+
+- Browse examples in the [HTTP documentation](/http/?cty=agent-cli) of other ways ngrok can augment your application
+- Bring your own [custom domain to ngrok](/http/#bring-your-own-domain)
+- Create connectivity to non-HTTP services such as SSH, RDP, or game servers using [TCP Endpoints](/tcp/)
+- Embed ngrok in your app programmatically without the agent with our [Agent SDKs](/agent-sdks/)
+- Add your [API key and automate via the `ngrok api` command](/api/#quickstart)
```

Comment:
I'm trying to find where this is implemented, but the next link in the nav at the bottom of the page links back to itself.